lib/models/project.js

```diff
@@ -52,6 +52,7 @@ class Project {
     this.setupNodeModulesPath();
     this.addonDiscovery = new AddonDiscovery(this.ui);
     this.addonsFactory = new AddonsFactory(this, this);
+    this.configCache = new Map();
```

Comment:
Can you make this `_configCache`?  We don't want it to be considered public API on `Project`.